# code_lab_work
R/bash code for lab work

In DepMap folder, there are R codes for analyzing cell dependency data from DepMap and cellign. 
Example:
![alt text](https://github.com/zzgeng/code_lab_work/blob/main/example_pictures/scatter_CRIPSR_exp.png)
X-axis is the CRISPR index (cell dependency score) and Y-axis is the gene (BMI1) expression in different cell lines. 

In scRNA-seq folder, there are code for single-cell RNA-seq clustering using Seurat, pseudotime analysis using monocle3 and code to plot modified violin plot. 
Example:
![alt text](https://github.com/zzgeng/code_lab_work/blob/main/example_pictures/violin_plot.png)

In bash script folder, there are bash script examples for HPC and slurm for ChIP-seq processing, peak calling and RNA-seq processing using kallisto. 
